clang/include/clang/Basic/Features.def

```diff
@@ -313,6 +313,8 @@ EXTENSION(datasizeof, LangOpts.CPlusPlus)
 
 FEATURE(cxx_abi_relative_vtable, LangOpts.CPlusPlus && LangOpts.RelativeCXXABIVTables)
 
+FEATURE(atomic_attributes, true)
```

Comment:
```suggestion
FEATURE(clang_atomic_attributes, true)
```

Many of these are c++  standard options, so I think we should be better about making sure we don't choose something that we'd want to use if the C/C++ standards introduced something similar.

Reply:
will change